tests/FSharp.Compiler.ComponentTests/EmittedIL/ComputationExpressions/ComputationExpressions.fs

```diff
@@ -73,3 +73,31 @@ module ComputationExpressions =
         compilation
         |> getCompilation
         |> verifyCompilation 
+
+    [<Theory; FileInlineData("CustomCollectionBuilderComputationExpr.fs", Optimize=BooleanOptions.False)>]
```

Comment:
This next two test cases can be combined if you want.
    [<Theory; FileInlineData("CustomCollectionBuilderComputationExpr.fs", Optimize=BooleanOptions.Both)>]
    let ``CustomCollectionBuilderComputationExpr_fs_OptimizeOff`` compilation =
        compilation
        |> getCompilation
        |> asExe
        |> withReferences [
            FsFromPath (Path.Combine (__SOURCE_DIRECTORY__,  "CustomCollectionBuilderComputationExprLibrary.fs"))
            |> withName "CustomCollectionBuilderComputationExprLibrary"
        ]
        |> withEmbeddedPdb
        |> withEmbedAllSource
        |> verifyILBaseline